.werft/jobs/build/prepare.ts

```diff
@@ -17,8 +18,23 @@ export async function prepare(werft: Werft) {
         exec("gcloud auth configure-docker --quiet");
         exec("gcloud auth configure-docker europe-docker.pkg.dev --quiet");
         exec('gcloud container clusters get-credentials core-dev --zone europe-west1-b --project gitpod-core-dev');
-        werft.done('prep');
+
+        if (lookForBranchNameWithVM(config.repository.branch)) {
```

Comment:
When you move this to job-config.ts please also log a line that says that we're overriding the value ☺️